Client/src/components/contanier/student/StudentCourses/index.js

```diff
@@ -1,11 +1,18 @@
 import React, { Component } from 'react'
+import data from './staticData'
+import Course from '../../../common/Course'
 import './index.css'
 
-export default class MyCourses extends Component {
+export default class StudentCourses extends Component {
   render() {
     return (
-      <div>
-        
+      <div className='studentcourses'>
```

Comment:
the class name should be 
`studentCourses`
or 
`student-courses`

Reply:
Done all edits